src/TrkAnaTreeMaker_module.cc

```diff
@@ -156,11 +157,15 @@ namespace mu2e {
         fhicl::Atom<art::InputTag> crvStepsTag{Name("CrvStepsTag"), Comment("Tag for CrvStep Collection"), art::InputTag()};
         fhicl::Atom<art::InputTag> crvDigiMCsTag{Name("CrvDigiMCsTag"), Comment("Tag for CrvDigiMC Collection"), art::InputTag()};
         fhicl::Atom<art::InputTag> crvDigisTag{Name("CrvDigisTag"), Comment("Tag for CrvDigi Collection"), art::InputTag()};
-        // CRV -- flags
+	// CRV -- flags
         fhicl::Atom<bool> crvhits{Name("FillCRVHits"),Comment("Flag for turning on crv CoincidenceClusterbranches"), false};
         fhicl::Atom<bool> crvpulses{Name("FillCRVPulses"),Comment("Flag for turning on crvpulseinfo(mc), crvwaveforminfo branches"), false};
         // CRV -- other
         fhicl::Atom<double> crvPlaneY{Name("CrvPlaneY"),2751.485};  //y of center of the top layer of the CRV-T counters.  This belongs in KinKalGeom as an intersection plane, together with the rest of the CRV planes FIXME
+        //Calorimeter -- input tag
+	fhicl::Atom<art::InputTag> caloClusterRecoTag{Name("CaloClusterRecoTag"), Comment("Tag for CaloClusterCollection"), art::InputTag()};
+	//Calorimeter -- flag
+	fhicl::Atom<bool> caloinfo{Name("FillCaloClusterInfo"), Comment("Flag for turning on Calorimeter cluster info"), true};
```

Comment:
Could you set the default to false, please? Then you can make your fcl file like:

```
#include "TrkAna/fcl/TrkAnaReco.fcl"
physics.analyzers.TrkAna.FillCaloClusterInfo : true
```

 We can discuss making ```true``` the default when we understand the impact on e.g. file sizes.